api/tests/conftest.py

```diff
@@ -37,31 +37,31 @@ def mock_detection_response():
         "box": {
             "name": "117319856-fc35bf00-ae8b-11eb-9b51-ca5aba673466.jpg",
             "geometries": [
-                [0.724609375, 0.1787109375, 0.7900390625, 0.2080078125],
-                [0.6748046875, 0.1796875, 0.7314453125, 0.20703125],
+                [0.8176307908857315, 0.1787109375, 0.9101580212741838, 0.2080078125],
+                [0.7471996155154171, 0.1796875, 0.8272978149561669, 0.20703125],
             ],
         },
         "poly": {
             "name": "117319856-fc35bf00-ae8b-11eb-9b51-ca5aba673466.jpg",
             "geometries": [
                 [
-                    0.7873152494430542,
+                    0.9063061475753784,
```

Comment:
comparison with ocr_predictor:

```
"words": [
    {
        "value": "Hello",
        "geometry": [0.7471996155154171, 0.1796875, 0.8272978149561669, 0.20703125],
        "confidence": 1,
        "crop_orientation": {"value": 0, "confidence": None},
    },
    {
        "value": "world!",
        "geometry": [
            0.8176307908857315,
            0.1787109375,
            0.9101580212741838,
            0.2080078125,
        ],
        "confidence": 1,
        "crop_orientation": {"value": 0, "confidence": None},
    },
],
```

```
"words": [
    {
        "value": "Hello",
        "geometry": [
            0.7453157305717468,
            0.1800299733877182,
            0.8233299851417542,
            0.17740298807621002,
            0.8250390291213989,
            0.2027825564146042,
            0.7470247745513916,
            0.20540954172611237,
        ],
        "confidence": 0.99,
        "crop_orientation": {"value": 0, "confidence": 1},
    },
    {
        "value": "world!",
        "geometry": [
            0.8157618045806885,
            0.18002046644687653,
            0.9063061475753784,
            0.17740710079669952,
            0.9078840017318726,
            0.20474515855312347,
            0.8173396587371826,
            0.20735852420330048,
        ],
        "confidence": 1,
        "crop_orientation": {"value": 0, "confidence": 1},
    },
],
```

Reply:
Why `geometry` changed so much ?

Reply:
@odulcy-mindee this was wrong before because remove padding was only in ocr_predictor (after detection predictor) 

Now it's the same as with the ocr predictor :)

Reply:
ref.: #1619 #1622  :)

Reply:
It's clearer, thank you